wp-content/themes/great-outdoor/header.php

```diff
@@ -54,49 +63,60 @@
 		endforeach;
 		?>
 
-		<!-- if Paddle page or Paddle products page, show promotion -->
-		<?php if ( $page_slug == 'paddle' || in_array( 'paddle', $product_cats ) ) : ?>
-			<!-- if promotion fields are set, add promo markup -->
-			<?php if ( isset( $promotion, $promotion_cta ) ) : ?>
+		<?php if ( in_array( 'paddle', $product_cats ) ) : ?>
+			<?php if ( isset( $cta ) ) : ?>
+				<aside class="paddle-cta">
```

Comment:
Should we use an include for this, to keep the markup encapsulated?

Reply:
Completely agree with you. This is has been contained in its own template file now.